bukkit/src/main/java/com/leonardobishop/quests/bukkit/command/AdminItemsCommandHandler.java

```diff
@@ -34,6 +35,7 @@ public void handle(CommandSender sender, String[] args) {
             }
             sender.sendMessage(ChatColor.GRAY.toString() + plugin.getQuestItemRegistry().getAllItems().size() + " items imported.");
             sender.sendMessage(ChatColor.DARK_GRAY + "Import a new held item using /q a items import <id>.");
+            sender.sendMessage(ChatColor.DARK_GRAY + "Give a quest item to a player using /q a items give <PlayerName> <id> <amount>.");
```

Comment:
```suggestion
            sender.sendMessage(ChatColor.DARK_GRAY + "Give a quest item to a player using /q a items give <player> <id> <amount>.");
```
Simple casing change just to match the rest of the plugin :)